python/tidy/servo_tidy/tidy.py

```diff
@@ -147,13 +143,47 @@ def filter_files(start_dir, only_changed_files, progress):
         yield file_name
 
 
+def uncomment(line):
+    for c in COMMENTS:
+        if line.startswith(c):
+            if line.endswith("*/"):
+                return line[len(c):(len(line) - 3)].strip()
+            return line[len(c):].strip()
+
+
+def licensed_mpl(header):
+    return licenses.MPL.replace('\n', ' ') in header
+
+
+def licensed_apache(header):
+    if licenses.APACHE.replace('\n', ' ') in header:
+        for c in licenses.COPYRIGHT:
+            if c in header:
+                return True
+    return False
```

Comment:
I disagree about silently returning `None`; it makes more sense to always return a boolean.

As for the array comprehension... I suppose you meant `return any(c in header for c in COPYRIGHT)`.